src/Action.php

```diff
@@ -9,8 +9,8 @@
  */
 class Action
 {
-    private $id;
-    private $destinationSceneId;
+    protected $id;
+    protected $destinationSceneId;
```

Comment:
An alternative way to do this would be to use the magic methods __sleep()/__wakeup() in which you should be able to serialzie private properties as well.